compiler/rustc_error_codes/src/error_codes/E0625.md

```diff
@@ -0,0 +1,27 @@
+Static and const variables can refer to other const variables. But a const
+variable cannot refer to a thread-local static variable.
```

Comment:
Generally the first sentence is just saying something very short. In this case it would be something like "A const variable is referring to a thread-local static variable".

Then you provide the full explanation below.